app/src/layouts/organizationsControl/organizationsPopover/organizationsPopover.jsx

```diff
@@ -57,6 +58,8 @@ export const OrganizationsPopover = ({ closePopover, closeSidebar }) => {
   const [isSearchEventTriggered, setIsSearchEventTriggered] = useState(false);
   const { trackEvent } = useTracking();
   const maxHeightPopover = window.innerHeight - MARGIN_TOP_AND_MARGIN_BOTTOM;
+  const location = useSelector(locationSelector);
+  const isAllOrganizationPage = location.pathname === '/organizations';
 
```

Comment:
Can we use this route as a constant to link with one defined in routes map?

Reply:
Done.